test/fixtures/categories.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
I've added some test files called "fixtures" that contain fake category and resource data. These do not exist in your development database, but are used in a separate _test_ database for automated testing.